package.json

```diff
@@ -24,7 +24,9 @@
     "react-native": "https://github.com/expo/react-native/archive/sdk-34.0.0.tar.gz",
     "react-native-dotenv": "^0.2.0",
     "react-native-gesture-handler": "~1.3.0",
+    "react-native-paper": "^2.16.0",
     "react-native-web": "^0.11.4",
+    "react-native-webview": "^7.0.5",
```

Comment:
Hey @Chisom, could elaborate the reason you're using this package? I believe that React-Native itself has a [WebView](https://facebook.github.io/react-native/docs/webview) component which can help out. Please let me know why you opt to use the package instead or/and we replace it with a component.

Reply:
The built-in WebView component was referring me to this package cause according to it, it will not be added to the react-native core in order to make it lighter.
Check out the  [read me](https://github.com/react-native-community/react-native-webview/blob/master/README.md)